.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Build
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Setup miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          channels: conda-forge,ome
+          environment-file: environment.yml
+          python-version: 3.9
+
+      - name: Install dependencies (temp)
+        shell: bash -l {0}
+        run: |
+          python -m pip install \
+          git+https://github.com/joshmoore/ome-zarr-py.git@nested
```

Comment:
similar to benchmark repo, is the branch ready for review/merge into mainline

Reply:
cc: @will-moore